SoundReplacer/Plugin.cs

```diff
@@ -19,6 +19,7 @@ public Plugin(Logger logger, Config config, Zenjector zenjector)
             Log = logger;
             Config = config.Generated<PluginConfig>();
             zenjector.UseLogger(logger);
+            zenjector.Install<AppInstaller>(Location.App);
```

Comment:
i didn't even know/remember that sira lets you target specific installers, but this doesn't even appear to work. that's probably because it's a patch and it explicitly needs to happen before, but all installers get called in the same phase.

how about moving good/bad cut patches into one and renaming it like this?
```c#
zenjector.Install<CutSoundsPatchesInstaller>(Location.App);
```

Reply:
honestly i'd just leave it as-is since there's no harmony in the mod anyway and it fits in. don't know if there's any real concerns.

as for the posfix attr. i'm aware of the default but i just prefer noting down the type of the patch somewhere, either by that or in the method name.

Reply:
I just took the habit of not specifying the attribute for Affinity patches unless it's not a Postfix. I don't mind keeping it as an App installer, what bothers me is that bad cut and good cut patches are split which doesn't make sense to me. I don't think you can inject `NoteCutSoundEffectManager` outside the game scene though.

Reply:
it makes sense to me in the sense that those systems are completely different functionality wise. slice sounds play regardless of when the note is hit - they always play on time. sometimes, even when you're not even remotely close to hitting the note, they still play. bad cut sounds are tied to the note however; you can even get a slice sound to play for a note then trigger its bad cut sound after :)